app/Http/Controllers/Mship/Management.php

```diff
@@ -264,7 +264,7 @@ public function getVerifyEmail($code)
 
     public function requestCertCheck()
     {
-        $ranRecently = ! Cache::add('USER_REQUEST_CERTCHECK_'.Auth::user()->id, '1', 1 * 60 * 60 * 1000); // 1 hour
+        $ranRecently = ! Cache::add('USER_REQUEST_CERTCHECK_'.Auth::user()->id, '1', 1 * 60 * 60); // 1 hour
```

Comment:
```suggestion
        $ranRecently = ! Cache::add('USER_REQUEST_CERTCHECK_'.Auth::user()->id, '1', now()->addHour()); // 1 hour
```
Something like (e.g. passing in a carbon instance) may be clearer to avoid similar confusion in future?